doc/modules/clustering.rst

```diff
@@ -523,13 +523,14 @@ below.
 ================================  ================================  ================================
 
 .. topic:: References:
-       
+
  * `"Multiclass spectral clustering"
    <https://www1.icsi.berkeley.edu/~stellayu/publication/doc/2003kwayICCV.pdf>`_
    Stella X. Yu, Jianbo Shi, 2003
 
- * :doi:`"Simple, direct, and efficient multi-way spectral clustering"<10.1093/imaiai/iay008>`
+ * :doi:`"Simple, direct, and efficient multi-way spectral clustering"
     Anil Damle, Victor Minden, Lexing Ying, 2019
+    <10.1093/imaiai/iay008>`
```

Comment:
To fix the sphinx warning
```suggestion
 * :doi:`"Simple, direct, and efficient multi-way spectral clustering"
   Anil Damle, Victor Minden, Lexing Ying, 2019
   <10.1093/imaiai/iay008>`
```

Reply:
Thank you very much for fixing this! If you have time, can you pls explain why this caused a problem?

Reply:
The lack of indentation generated a sphinx warning during the [rendering](https://178390-843222-gh.circle-artifacts.com/0/doc/_changed.html).
Note that you can access the rendering of your modification clicking on the 'Details' on the doc-artifacts check

![image](https://user-images.githubusercontent.com/1662261/156050231-eacb9288-8b30-4e8e-9f5d-c32b788a8038.png)

Reply:
Yes that (and also the warning) I figured out myself, but that the warning was refering to this, baffles me.